web/src/beta/components/TabMenu/index.tsx

```diff
@@ -0,0 +1,83 @@
+import { FC, ReactNode } from "react";
+
+import Icon from "@reearth/beta/components/Icon";
+import Icons from "@reearth/beta/components/Icon/icons";
+import Text from "@reearth/beta/components/Text";
+import { styled, useTheme } from "@reearth/services/theme";
+
+interface TabObject {
+  icon: keyof typeof Icons;
+  component: ReactNode;
+  key: string;
+}
+
+type Props = {
+  title: string;
+  tabs: TabObject[];
+  selectedTab: string;
+  onSelectedTabChange: (tab: string) => void;
+};
+
+const TabMenu: FC<Props> = ({ tabs, title, selectedTab, onSelectedTabChange }) => {
+  const theme = useTheme();
+
+  const selectedTabItem = tabs.find(({ key }) => key === selectedTab);
+
+  return (
+    <Wrapper>
+      <Title>
+        <Text size="body">{title}</Text>
+      </Title>
```

Comment:
We don't need this. Please look at the left/right panel implementations we already have to see how they are built up. 
Reference `src/beta/features/Editor/tabs/widgets/SidePanel/index.tsx` to see how it should be used (it'd be passed as the `children` prop)

Reply:
Changed the implementation. Thanks for this. 
Waiting for you to resolve the other comments on this!